src/main/res/values/strings.xml

```diff
@@ -139,23 +139,23 @@
   <string name="unknown_error_description">Unknown error.</string>
   <string name="unknown_format">Unknown: %s</string>
   <string name="unsigned">Unsigned</string>
-  <string name="unsigned_description">Unsigned. The list of applications could not be verified. Be careful with
-    applications downloaded from unsigned repositories.</string>
+  <string name="unsigned_description">Unsigned. Could not verify the application list. Be careful downloading
+    applications from unsigned repositories.</string>
   <string name="unstable_updates">Unstable updates</string>
-  <string name="unstable_updates_summary">Suggest updates to unstable versions</string>
+  <string name="unstable_updates_summary">Suggest installing unstable versions</string>
   <string name="unverified">Unverified</string>
   <string name="update">Update</string>
-  <string name="update_notifications">Update notifications</string>
-  <string name="update_notifications_summary">Show a notification when updates are available</string>
+  <string name="update_notifications">Notifications about newer application versions</string>
+  <string name="update_notifications_summary">Announce applications awaiting installation</string>
   <string name="updates">Updates</string>
   <string name="updates_empty_summary">All applications up to date</string>
   <string name="username">Username</string>
-  <string name="username_is_not_specified">Username is not specified</string>
-  <string name="validation_index_error_description">Failed to validate the index.</string>
-  <string name="validation_package_error_description_format">Failed to validate the package: %s.</string>
-  <string name="validation_error_hash_lower">invalid hash</string>
-  <string name="validation_error_integrity_lower">integrity check failed</string>
-  <string name="validation_error_permissions_lower">missing permissions</string>
+  <string name="username_is_not_specified">Username missing</string>
+  <string name="validation_index_error_description">Index could not be validated:</string>
+  <string name="validation_package_error_description_format">APK package could not be validated: %s.</string>
+  <string name="validation_error_hash_lower">invalid checksum</string>
+  <string name="validation_error_integrity_lower">could not check integrity</string>
+  <string name="validation_error_permissions_lower">the application metadata permissions are wrong</string>
```

Comment:
Not sure about this

Reply:
The most important problem with this string is that it's too long ("especially with APK package could not be validated"). Use "Package could not be validated" and something short instead of "the application metadata permissions are wrong" ("missing permissions" was pretty fine, "invalid permissions" or something like that can be used instead if you don't like "missing" here). The full string will be "Package could not be validated: missing permissions" or "Package could not be validated: invalid permissions".

Reply:
I don't see how that is descriptive, or how it follows the earlier explanation, but will revert.